MeetupPlatformApi/Authentication/AuthenticationManager.cs

```diff
@@ -21,27 +21,43 @@ public CurrentUserInfo GetCurrentUserInfo(ClaimsPrincipal user)
         var idClaim = user.Claims.Single(claim => claim.Type == ClaimTypes.NameIdentifier);
         var id = Guid.Parse(idClaim.Value);
 
-        return new() {UserId = id};
+        return new() { UserId = id };
     }
 
-    public string IssueAccessToken(UserEntity user) =>
-        IssueToken(
+    public TokenPair IssueTokenPair(UserEntity user)
+    {
+        var accessToken = IssueAccessToken(
             payload: new Dictionary<string, object>
-            {
-                {ClaimTypes.NameIdentifier, user.Id}
-            },
-            lifetime: configuration.AccessTokenLifetime);
+                {
+                    {ClaimTypes.NameIdentifier, user.Id}
+                },
+                accessTokenLifetime: configuration.AccessTokenLifetime);
+
+        var refreshToken = IssueRefreshToken(configuration.RefreshTokenLifetime, user.Id);
 
-    private string IssueToken(IDictionary<string, object> payload, TimeSpan lifetime)
```

Comment:
I don't think, that You should get rid from this method, since it would come in handy for generating both Access and Refresh tokens.